src/tests/resistance/fuzzing_incorrect_checksum.rs

```diff
@@ -368,8 +368,10 @@ fn encode_messages_and_corrupt_checksum(
         .map(|_| {
             let message = message_pool.choose(rng).unwrap();
 
-            let mut buffer = vec![];
-            let mut header = message.encode(&mut buffer).unwrap();
+            let mut body_buffer = Vec::new();
+            let mut header = message.encode(&mut body_buffer).unwrap();
+
+            let mut buffer = Vec::with_capacity(body_buffer.len() + 24);
```

Comment:
fyi, there's a constant for various message properites including header length here: https://github.com/eqlabs/ziggurat/blob/master/src/protocol/message/constants.rs